components/service/firefox-accounts/src/test/java/mozilla/components/service/fxa/FxaAccountManagerTest.kt

```diff
@@ -398,13 +398,15 @@ class FxaAccountManagerTest {
         assertNotNull(latestSyncManager!!.dispatcher.inner)
         verify(latestSyncManager!!.dispatcher.inner, times(1)).startPeriodicSync(any(), anyLong())
         verify(latestSyncManager!!.dispatcher.inner, never()).stopPeriodicSync()
-        verify(latestSyncManager!!.dispatcher.inner, times(1)).syncNow(anyBoolean())
+        verify(latestSyncManager!!.dispatcher.inner, times(1)).syncNow(anyBoolean(), anyBoolean())
 
         // Can trigger syncs.
         manager.syncNowAsync().await()
-        verify(latestSyncManager!!.dispatcher.inner, times(2)).syncNow(anyBoolean())
+        verify(latestSyncManager!!.dispatcher.inner, times(2)).syncNow(startup = false, debounce = false)
         manager.syncNowAsync(startup = true).await()
-        verify(latestSyncManager!!.dispatcher.inner, times(3)).syncNow(anyBoolean())
+        verify(latestSyncManager!!.dispatcher.inner, times(1)).syncNow(startup = true, debounce = false)
+        manager.syncNowAsync(startup = true, debounce = true).await()
+        verify(latestSyncManager!!.dispatcher.inner, times(1)).syncNow(startup = true, debounce = true)
```

Comment:
@grigoryk Just to confirm, is this verifying #3284? Or do we need an extra test for it?

Reply:
We don't have tests covering dispatcher/worker behaviour right now, unfortunately. So this is _partially_ verifying that, but not quite :)

Reply:
👍